h2o-r/h2o-package/R/models.R

```diff
@@ -2584,6 +2584,10 @@ h2o.get_regression_influence_diagnostics <- function(model, predictorSize = -1)
 #' }
 #' @export 
 h2o.negative_log_likelihood <- function(model) {
```

Comment:
please add the flag check as well and change the call from model.model_performance().loglikelihood() to h2o.likelihood(modelName).  Same for aic().

if (model@allparameters$calc_like) {
  warning_message <- paste("This is the simplified negative log likelihood function used during training for speedup.",
                              "To see the correct values (for loglikelihood and AIC), set calc_like=True and call", 
                              "h2o.loglikelihood(model) and h2o.aic(model)")
}  else {
  warning_message <- paste("This is the simplified negative log likelihood function used during training for speedup.",
                              "To see the correct values (for loglikelihood and AIC), set calc_like=True, retrain and call", 
                              "h2o.loglikelihood(model) and h2o.aic(model)")
}